src/core/js/scriptLoader.js

```diff
@@ -30,7 +30,8 @@
                 a11y: 'libraries/jquery.a11y',
                 scrollTo: 'libraries/scrollTo',
                 bowser: 'libraries/bowser',
-                'enum': 'libraries/enum'
+                enum: 'libraries/enum',
+                jqueryMobile: 'libraries/jquery.mobile.custom'
```

Comment:
when did I mention that? I can't remember now lol. I'm all for adding touchpunch as well. In fact I vaguely remember experiencing issues when I included it manually on two different plugins so having it loaded once on the document would solve this. Probably a good idea - especially if Adapt is going more and more in this direction.